app/models/eventParticipant.py

```diff
@@ -5,7 +5,7 @@
 class EventParticipant(baseModel):
     user = ForeignKeyField(User)
     event = ForeignKeyField(Event, backref="participants")
-    hoursEarned = FloatField(null=True)
+    hoursEarned = FloatField(null = False, default = 0)
```

Comment:
@BrianRamsay Will this leave problems elsewhere? Are we using `Null` checks elsewhere in code?

Reply:
A good catch. There's a check in `app/templates/events/manageVolunteers.html` that is no longer relevant.